### PR TITLE
Xendiz bid adapter

### PR DESCRIPTION
**Pull request by dimashirokov**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'xendiz',
  params: {
    pid: '00000000-0000-0000-0000-000000000000'
  }
}
```
- contact email of the adapter’s maintainer: dima@xendiz.com
- [x] official adapter submission

### CODE REVIEW

**Comment by mkendall07** [comment]
@dimashirokov 
Thanks for the PR. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html).

**Comment by mkendall07** [comment]
Testing this on a non-secure page doesn't seem to work. Getting a CORS error:
```Failed to load http://prebid.xendiz.com/request: Redirect from 'http://prebid.xendiz.com/request' to 'https://prebid.xendiz.com/request' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://ap.localhost:9999' is therefore not allowed access.```

File: modules/xendizBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse) {
+    const bids = serverResponse.body.bids.map(bid => {
+      return {
+        requestId: bid.id,
+        bidderCode: BIDDER_CODE,
```
Comment (mkendall07) [inline modules/xendizBidAdapter.js]: please drop this line. It's added automatically.

**Comment by dimashirokov** [comment]
Hello @matthewlane 
Thank you for review. I updated the code based on your comments, CORS issue solved too.
Here is docs pull request https://github.com/prebid/prebid.github.io/pull/566

**Pull request merged**